server/lib/pbench/test/conftest.py

```diff
@@ -24,7 +24,7 @@ def setup(request, pytestconfig):
 
 
 @pytest.fixture
-def test_logger(pytestconfig):
+def logger(pytestconfig):
```

Comment:
We haven't been doing this, and I'm not really sure we want to... but looking at the isolated diffs I had to skip back to check on the return value, and a return type hint would be handy. On the other hand, I have no idea how we'd want to document the "type" of the fixture parameters, which are resolved/overloaded pytest fixtures themselves... which we define as functions but at this point are just symbols bound to the fixture return values.

Or maybe we just ignore the parameter types for fixtures but still try to add a return value type hint?

Never mind ... not something to attack in this PR. 😀 🙄